.github/workflows/test.yaml

```diff
@@ -722,6 +724,55 @@ jobs:
         if: failure()
         run: bash .github/slack/notify-failure.sh ${{ github.job}} ${{ matrix.platform }}
 
+  data-proxy:
+    needs: start-time
+    timeout-minutes: 60 # can take longer if platforms are down, so better protect
+    strategy:
+      fail-fast: false
+      matrix:
+        platform:
+          - cloudflare-workers
+          - vercel-edge-functions
+        clientEngine: ['dataproxy']
+        os: [ubuntu-latest] #, windows-latest, macos-latest]
+    runs-on: ${{ matrix.os }}
+    concurrency: ${{ github.run_id }}-platforms-serverless-${{ matrix.platform }}
+    env:
+      START_TIME: ${{ needs.start-time.outputs.start-time }}
+      NPM_CONFIG_LOGLEVEL: error
+      NODE_ENV: development
+      NODE_MODULES_CACHE: false
+      NODE_VERBOSE: true
+      AES_256_PASS: ${{ secrets.AES_256_PASS }}
+      PRISMA_CLIENT_ENGINE_TYPE: dataproxy
```

Comment:
Should this potentially be dynamic via `matrix.clientEngine`?

Reply:
I'd probably rename this test folder to something like `v8` or `dataproxy-v8`. In this case, I just want to test on the two very specific serverless platforms that are `cloudflare-workers` and `vercel-edge-functions`. I think that it would make sense to flip that switch on the other workflows? Before doing that, I'd like to have a way to init a dataproxy with our usual `DATABASE_URL`.

Reply:
I don't think I understand you answer. But if it works, this is perfectly fine of course.

Reply:
Yep, sorry. The confusion came from my side! I'll exclude `binary` and `library` engines for these two tests and put them back into `platforms-serverless`. Thanks!